### ISSUE TITLE
No active checkers for PHP

### ISSUE THREAD

**Issue by tarciozemel** [scrooloose/syntastic#780]
Not so much time ago I'm was using Syntastic very well in PHP files. Unfortunately, I had to do a new fresh **Vim 7.3.584** instalation and put all my plugins to run again. With my `.vimrc` backup restored and running the command to Vundle install all the stuff, was easy (and it installed the latest version of Syntastic, of course)!

Unfortunately, after that, Syntastic isn't work anymore... When I open a PHP file and type `:SyntasticInfo`, I get

```
Syntastic info for filetype: php
Available checkers: php phpcs phpmd
Currently active checker(s):
```

In other words, the checkers are available but Syntastic is not activating them in the files, since "Currently active checker(s)" is empty...

What do you recommends me to do?
